### PR TITLE
sql: add description for statistics

### PR DESCRIPTION
**Pull request by lamxTyler**

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: sql/introduction-to-statistics.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ANALYZE TABLE TableNameList
+> 该语句会收集 TableNameList 中所有表的统计信息。
+
+ANALYZE TABLE TableName INDEX IndexNameList
+> 该语句会收集 TableName 中所有的 IndexNameList 中的索引列的统计信息。
+```
+
+### 自动更新
+
+在发生增加，删除以及修改语句时，TiDB 会自动更新表的总行数以及修改的行数。这些信息会定期持久化下来，更新的周期可以由 `statsLease` 参数指定，
```
Comment (andelf) [inline sql/introduction-to-statistics.md]: ....why an upper case L here?
Should TiDB fix this first?

Comment (coocood) [reply sql/introduction-to-statistics.md]: In the config file, this has changed to `stats-lease`.

**Comment by queenypingcap** [comment]
@andelf PTAL.

**Comment by shenli** [comment]
introduction-to-statistics.md is too long. statistics.md is enough.

**Comment by shenli** [comment]
LGTM

**Pull request merged**